FeedStoreChallenge/ManagedFeedImage.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  ManagedFeedImage.swift
+//  FeedStoreChallenge
+//
+//  Created by Vladimir Jeremic on 4/27/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(ManagedFeedImage)
+public class ManagedFeedImage: NSManagedObject {
```

Comment:
This class is not going to be overridden thus it can be final.
Also it is a `CoreDataFeedStore` implementation detail so it should not be public

Please make it final and internal.

```suggestion
final class ManagedFeedImage: NSManagedObject {
```